src/main/commands/scan.yml

```diff
@@ -4,6 +4,10 @@ parameters:
     description: the name of the environment variable where the SonarCloud API token is stored 
     default: SONAR_TOKEN
     type: env_var_name
+  sonar_config_path:
+    description: path to the sonnar-scanner config to use (e.g. ~/my-repo-configs/.sonarcloud.properties)
```

Comment:
`.sonarcloud.properties` indicate that you are using Automatic Analysis mode. In this context we are using a CI, so it would be best to not give `.sonarcloud.properties` as the example file name but `sonar-project.properties`, which is the convention for CI analysis.